tests/test_quantum_gradients.py

```diff
@@ -465,13 +467,13 @@ def d_error(p, grad_method):
             ret = 0
             for d_in, d_out in zip(in_data, out_data):
                 args = np.array([d_in, p])
-                diff = (classifier(args) - d_out)
+                diff = (classifier(*args) - d_out)
```

Comment:
same question

Reply:
The classifier was defined as `classifier_circuit(in_data, x):`, so `args = np.array([d_in, p])` should be unpacked before being passed to the function. Same answer as before - test should not have been passing, don't know why it was previously :thinking: